src/index.js

```diff
@@ -203,17 +206,19 @@ const deriveNodeKeys = (mnemonic, passphrase) => {
  *
  * @param  {Uint8Array}  master a master seed
  * @param  {String}  type one of 'ownership', 'transfer', 'spawn', 'voting',
- *   'management'
+ *   'management', 'bitcoin'
```

Comment:
```suggestion
 *                   'management', 'bitcoin'
```

Reply:
Realizing now this was already ugly prior to your changes, just like the rest of these doccomments. Do as you wish!

Reply:
Fixed them all